README.md

```diff
@@ -63,178 +72,377 @@ bambuAnnotations <- prepareAnnotations(gtf.file)
-    + counts: expression estimates
-    + CPM: sequencing depth normalized estimates
-    + fullLengthCounts: estimates of read counts mapped as full length reads for each transcript
-    + partialLengthCounts: estimates of read counts mapped as partial length reads for each transcript
-    + uniqueCounts: counts of reads that are uniquely mapped to each transcript
-    + theta: raw estimates
-* access annotations that are matched to the transcript expression estimates by ***rowRanges()***
-* access transcript to gene id map by ***rowData()***, *eqClass* that defines the equivalent class transcripts is also reported
+### De-novo transcript discovery
 
-In the case when *quant* is set to FALSE, i.e., only transcript discovery is performed, 
-***bambu*** will report the *grangeslist* of the extended annotations
+In cases where the organism does not yet have reference annotations, or unreliable annotations, bambu can be run in de-novo mode. In de-novo mode, bambu does not train a model, and instead uses the pretrained model to classify novel transcripts (see [Using a pretrained model](#Using-a-pretrained-model). To learn how to train a new model for a more closely related organism/sample see [Training a model on another species/dataset and applying it](#Training-a-model-on-another-speciesdataset-and-applying-it). Without annotations bambu is unable to calibrate the NDR output, nor be able to recommend a threshold and will instead use the TPS as the thresholded value. Therefore you should supply a manual NDR threshold ([Modulating the sensitivity of discovery (pre and post analysis)](#Modulating-the-sensitivity-of-discovery-pre-and-post-analysis)) and note that the precision of the output is unlikely to linearly match an applied threshold.
+The TPS threshold used is (> 1-NDR). If an NDR is not provided, a default NDR threshold of <0.1 is used (an effective TPS threshold of > 0.9). As in  [Modulating the sensitivity of discovery (pre and post analysis)](#Modulating-the-sensitivity-of-discovery-pre-and-post-analysis) an NDR of 1 can be provided to output all possible read classes with their TPS scores 
 
-### Complementary functions
+```rscript
+novelAnnotations <- bambu(reads = test.bam, annotations = NULL, genome = fa.file, NDR = 0.5, quant = FALSE)
+```
+### Storing and using preprocessed files (rcFiles)
 
-**Transcript expression to gene expression**
+The first step of *bambu* involves the construction of read classes which is a large fraction of the running time. This could be time-consuming if we want to perform transcript discovery & quantification multiple times on the same dataset using different configurations (eg. NDR, or combinations of samples), especially when the sample is large. To mitigate this, we can store the read class information as read class files (rcFiles) during a bambu run. Then they can be used as an input argument in the  bambu main function for the subsequent bambu run.
```

Comment:
```suggestion
The first step of *bambu* involves the construction of read classes which is a large fraction of the running time. This could be time-consuming if we want to perform transcript discovery & quantification multiple times on the same dataset using different configurations (eg. NDR, or combinations of samples), especially when the sample is large. To mitigate this, we can store the read class information as read class files (rcFiles) during a *bambu* run. Then they can be used as an input argument in the  *bambu* main function for the subsequent *bambu* run.
```